### PR TITLE
Tag services with acts-as-taggable-on gem

### PR DESCRIPTION
**Pull request by c-lliope**
https://github.com/mbleigh/acts-as-taggable-on

This gives us a pretty flexible solution
for tagging organizations with services.

### CODE REVIEW

**Comment by c-lliope** [comment]
@rsiegel827 @averon I combined both of your commits into this one - feel free to push more commits if you like. Any thoughts on my changes?

File: config/routes.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@
   get "/organizations/sidebar", to: "organizations#sidebar"
   get "/organizations/:id/change_status", to: "organizations#change_status", as: :change_organization_status
   resources :organizations
+  resources :tags, only: [:index, :show]
```
Comment (c-lliope) [inline config/routes.rb]: Are we using this route to display tags? If not, can we remove it?

File: app/controllers/tags_controller.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class TagsController < ApplicationController
+  def index
+    @tags = ActsAsTaggableOn::Tag.all
+  end
+
+  def show
+    @tag = ActsAsTaggableOn::Tag.find(params[:id])
+    @organizations = Organization.tagged_with(@tag.name)
+  end
```
Comment (c-lliope) [inline app/controllers/tags_controller.rb:5]: same with this controller - are we using it at all?

Comment (robert-baldwin) [inline app/controllers/tags_controller.rb:5]: I believe so? #index is for getting all the tags for the sidebar list, and #show displays all the organizations for a particular tag.

**Comment by robert-baldwin** [comment]
:eyes: good overall! :+1: on specs. Maybe we don't need the tags controller (I may be missing something). See you this afternoon!

**Pull request merged**